### ISSUE TITLE
Training recopies for the Weakly Supervised Pre-training paper

### ISSUE THREAD

**Issue by netw0rkf10w** [facebookresearch/ClassyVision#790]
## 📚 Documentation
Hello,

In the README you mentioned the following: 

> It offers several notable advantages:
> 
> ...
> - Demonstrated success in training at scale. We’ve used it to replicate the state-of-the-art results from the paper [Exploring the Limits of Weakly Supervised Pretraining](https://arxiv.org/abs/1805.00932).
> ...

I would like to know if you could share the training recipes (*.json) for the results that you were able to replicated.

Thank you in advance.

Best regards.

**Comment by mannatsingh** [comment]
Hi @netw0rkf10w, unfortunately this was trained a couple of years ago so it's hard to track the configs down for this. I'm afraid we won't be able to share this. But the training logic was fairly similar to training on imagenet, except that there were multiclass labels normalized to sum to 1. Other than that, everything should match up to imagenet 1k training with a cross entropy loss that handles soft targets.

**Comment by mannatsingh** [comment]
Please feel free to reopen if you have more questions
